Gems/Atom/RHI/Code/Source/RHI/AliasedHeap.cpp

```diff
@@ -124,7 +124,8 @@ namespace AZ::RHI
         if (!CheckBitsAny(m_compileFlags, TransientAttachmentPoolCompileFlags::DontAllocateResources))
         {
             AZ_Assert(m_totalAllocations < m_cache.GetCapacity(),
-                "Exceeded the size of the cache! This will destroy actively used resources");
+                "Exceeded the size of the cache! This will destroy actively used resources. (Total Allocations was %i, and Cache capacity was %i)",
+                m_totalAllocations, m_cache.GetCapacity());
             m_totalAllocations++;
 
```

Comment:
Is the actual type of the return of m_totalAllocations and m_cacheGetCapacity signed integers, as in, %i ? 
I looked at https://github.com/o3de/o3de/blob/ecda54e0b5ce170824698e06ab96801149a825fd/Gems/Atom/RHI/Code/Include/Atom/RHI/ObjectCache.h#L78C32-L78C33
and it appears that its size_t, so I'm thinking that this will likely trigger warnings, which may be interpreted as errors.

Also, m_totalAllocations is of type `uint32_t m_totalAllocations = 0;`

perhaps to be safe, use 
```
%"PRIu32"
```
for the total allocations?

maybe Consider using %zu forthe GetCapcacity.?